git/git.go

```diff
@@ -733,15 +733,18 @@ func CloneWithoutFilters(flags CloneFlags, args []string) error {
 	// not working. You can get around that with https://github.com/kr/pty but that
 	// causes difficult issues with passing through Stdin for login prompts
 	// This way is simpler & more practical.
-	filterOverride := ""
+	filterDriverOverride := ""
```

Comment:
Empty variable initializations are typically done in go using the `var` block. I would write this like so:

```go
var (
        filterDriverOverride bool
        smudgeFilterOverride string
)
```

and then use the appropriate [`fmt` verbs](https://golang.org/pkg/fmt/) to turn the above `string` and `bool` into `cmdargs` down below 😄